### PR TITLE
Update Set-CsClsConfiguration.md

### PR DESCRIPTION
**Pull request by priyanshamishra**
We need the following documentation update for the cmdlet Set-CsClsConfiguration for a CU6 bug that will be shipped next month.

### CODE REVIEW

**Comment by priyanshamishra** [comment]
@MicrosoftDocs/marveldocs-pubops PR is ready for review. Please review and publish.

**Review by get-itips — COMMENTED** [review]
@priyanshamishra please add these parameters to the SYNTAX section of the article too

File: skype/skype-ps/skype/Set-CsClsConfiguration.md [hunk 1]
```diff
@@ -496,6 +496,105 @@ Accept pipeline input: False
...
+Required: False
+Position: Named
+Default value: 250000000000
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+
+### -ETLMaxRetentionInDays
+Maximum retention time in days for ETL files
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsClsConfiguration.md:583]:
```suggestion
Maximum retention time in days for ETL files.
```

File: skype/skype-ps/skype/Set-CsClsConfiguration.md [hunk 2]
```diff
@@ -496,6 +496,105 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: 400000000000
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ETLEnoughQuotaInBytes
+Enough ETL usage quota to make purge stop due max quota condition reached
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsClsConfiguration.md:566]:
```suggestion
Enough ETL usage quota to make purge stop due maximum quota condition reached.
```

File: skype/skype-ps/skype/Set-CsClsConfiguration.md [hunk 3]
```diff
@@ -496,6 +496,105 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: 3221225472
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ETLMaxQuotaInBytes
+Max allowed quota used by ETL files before purge starts
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsClsConfiguration.md:550]:
```suggestion
Maximum allowed quota used by ETL files before purge starts.
```

File: skype/skype-ps/skype/Set-CsClsConfiguration.md [hunk 4]
```diff
@@ -496,6 +496,105 @@ Accept pipeline input: False
...
+
+Required: False
+Position: Named
+Default value: 4000000000
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ETLMinQuotaInBytes
+Min quota allowed for ETL files, even if disk size is less than ETLMinFreeSpaceInDiskInBytes.
```
Comment (dariomws) [inline skype/skype-ps/skype/Set-CsClsConfiguration.md:534]:
```suggestion
Minimum quota allowed for ETL files, even if disk size is less than ETLMinFreeSpaceInDiskInBytes.
```

**Review by dariomws — APPROVED** [review]
I'm ok after @get-itips suggestions. I've also added some minor suggestions too.

**Comment by priyanshamishra** [comment]
> @priyanshamishra please add these parameters to the SYNTAX section of the article too

Updated.

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Review by tseward — APPROVED** [review]
Approve with dariomws' changes.

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by tiburd** [comment]
close > reopen

**Pull request merged**